.github/policies/untriaged.yml

```diff
@@ -0,0 +1,37 @@
+id: untriaged
+name: GitOps.PullRequestIssueManagement
+description: Manage the 'untriaged' label on issues
+owner:
+resource: repository
+disabled: false
+where:
+configuration:
+  resourceManagementConfiguration:
+    eventResponderTasks:
+    - if:
+      - payloadType: Issues
+      - isOpen
+      - isNotInAnyMilestone
+      - or:
+        - isAction:
+            action: Opened
+        - isAction:
+            action: Reopened
+      - isNotLabeledWith:
+          label: untriaged
+      then:
+      - addLabel:
+          label: untriaged
+      description: Add untriaged label to new/reopened issues without a milestone
+    - if:
+      - payloadType: Issues
+      - or:
+        - isAction:
+            action: Closed
+        - isPartOfAnyMilestone
```

Comment:
@JohannesLampel I suspect the check failure is caused by `IsPartOfAnyMilestone`, but I thought that condition existed (desite being absent in the docs).

Reply:
The `isNotInAnyMilestone` condition does not exist. Same goes for the `isNotLabeledWith` condition. However, you can use the `not` modifier to achieve what you want to achieve. I've added comments with suggested changes.